docs/content/user-guides/crd-acme/03-deployments.yml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: traefik-ingress-controller
+
```

Comment:
Since you referenced a namespace in the CRB, you need to add the namespace here to confirm

Reply:
Well, things as they are right now seem to be working. Whereas when I tried your suggestion, they started to break, so I probably misunderstood what you meant, and it's making my heard hurt right now. I'll have another pass at it later.

Reply:
done.